Sources/NIOHTTP2/HTTP2Error.swift

```diff
@@ -1690,7 +1694,7 @@ private func _location(file: String, line: UInt) -> String {
     return "\(file):\(line)"
 }
 
-private final class StringAndLocationStorage: Equatable {
+private final class StringAndLocationStorage: Equatable,  @unchecked Sendable {
```

Comment:
This shouldn't be `Sendable`, it's a class with mutable properties. It can be used in a `Sendable` type if the caller does the appropriate CoW dance though.